### PR TITLE
[portaudio] update to 19.7

### PR DESCRIPTION
**Pull request by Be-ing**
**Describe the pull request**

- #### What does your PR fix?
update to new upstream release

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
all

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
yes

### CODE REVIEW

File: ports/portaudio/fix-library-can-not-be-found.patch [hunk 1]
```diff
@@ -1,16 +1,16 @@
-diff --git a/CMakeLists.txt b/CMakeLists.txt
```
Comment (JonLiu1993) [inline ports/portaudio/fix-library-can-not-be-found.patch:1]: @Be-ing ,Please revert the format of patch

File: ports/portaudio/portfile.cmake [hunk 2]
```diff
@@ -1,43 +1,44 @@
-vcpkg_fail_port_install(ON_TARGET "uwp")
```
Comment (JonLiu1993) [inline ports/portaudio/portfile.cmake:1]: Please revert the format of patch

File: ports/portaudio/vcpkg.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "portaudio",
-  "version-date": "2020-02-20",
+  "version-string": "19.7",
```
Comment (JonLiu1993) [inline ports/portaudio/vcpkg.json:3]: Please change "version-string" to "version"

File: versions/p-/portaudio.json [hunk 4]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "9136721c237551198502ff36d244e176ebca3955",
+      "version-string": "19.7",
```
Comment (JonLiu1993) [inline versions/p-/portaudio.json:5]: Please change "version-string" to "version"

**Comment by Be-ing** [comment]
macOS CI failed and I don't understand why...

```
Starting: *** Test Modified Ports and Prepare Test Logs ***
==============================================================================
Task         : PowerShell
Description  : Run a PowerShell script on Linux, macOS, or Windows
Version      : 2.180.1
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/powershell
==============================================================================
Generating script.
========================== Starting Command Output ===========================
##[error]Unable to locate executable file: 'pwsh'. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.
Finishing: *** Test Modified Ports and Prepare Test Logs ***
```

**Comment by JonLiu1993** [comment]
@Be-ing ,This is not a problem of portaudio itself, it has been solved now, we wait for the result after rerun

**Comment by ras0219-msft** [comment]
LGTM, thanks for the PR!

**Pull request merged**